lib/puppet/provider/service/upstart.rb

```diff
@@ -1,5 +1,3 @@
-require 'semver'
```

Comment:
I was wondering if anyone else copy-pasted the `upstart` provider which requires `semver` but never uses it (or more likely used it at some point but that code's been removed). Do any other modules on the forge do that?